src/components/views/LocalModList/LocalModCard.vue

```diff
@@ -67,7 +96,13 @@ export default class LocalModCard extends Vue {
     viewDependencyList() {
         this.$emit('viewDependencyList', this.mod);
     }
+
+    created() {
+        this.updateDependencies();
+    }
 }
+
+const dependencyStringToModName = (x: string) => x.substring(0, x.lastIndexOf('-'));
```

Comment:
This should be declared as an explicit function rather than an anonymous one. Aware there's not a difference in terms of execution however it makes it clear initially that it's a function.

Just a personal preference.

Reply:
Fair enough, I can change it when/if #1202 is merged.